src/Sarif/Writers/ResultLogJsonWriter.cs

```diff
@@ -294,7 +294,7 @@ public void CompleteRun()
 
             SerializeIfNotNull(_run.Tags, "tags");
             SerializeIfNotNull(_run.Conversion, "conversion");
-            SerializeIfNotNull(_run.Language, "language");
+            SerializeIfNotDefault(_run.Language, "language", "en-US");
```

Comment:
>SerializeIfNotDefault [](start = 12, length = 21)

This is the fix for #1524, "ResultLogJsonWriter emits language 'en-US'". Note that I hard-coded the default -- this string appears in several places and I chose not to make today the day I DRY it out.